language/French/strings.po

```diff
@@ -43,305 +43,305 @@ msgstr ""
 msgid "Custom UI Settings"
-msgstr ""
+msgstr "Configurer l'Interface Personnalisé"
 
 #: src/emu/ui/custui.cpp:369
 msgid "UI Font"
-msgstr ""
+msgstr "Police de l'Interface"
 
 #: src/emu/ui/custui.cpp:373
 msgid "Bold"
-msgstr ""
+msgstr "Gras"
 
 #: src/emu/ui/custui.cpp:374
 msgid "Italic"
-msgstr ""
+msgstr "Italique"
 
 #: src/emu/ui/custui.cpp:380
 msgid "Lines"
-msgstr ""
+msgstr "Lignes"
 
 #: src/emu/ui/custui.cpp:388
 msgid "Infos text size"
-msgstr ""
+msgstr "Infos sur la taille du texte"
 
 #: src/emu/ui/custui.cpp:405
 msgid "UI Fonts Settings"
-msgstr ""
+msgstr "Paramètres des Polices de l'Interface"
 
 #: src/emu/ui/custui.cpp:432
 msgid "Sample text - Lorem ipsum dolor sit amet, consectetur adipiscing elit."
-msgstr ""
+msgstr "Texte Echantillon - Lorem ipsum dolor sit amet, conestetur adipiscing elit"
 
 #: src/emu/ui/custui.cpp:530
 msgid "Normal text"
-msgstr ""
+msgstr "Texte Normal"
 
 #: src/emu/ui/custui.cpp:531
 msgid "Selected color"
-msgstr ""
+msgstr "Couleur Sélectionné"
 
 #: src/emu/ui/custui.cpp:532
 msgid "Normal text background"
-msgstr ""
+msgstr "Texte Normal en Arrière Plan"
 
 #: src/emu/ui/custui.cpp:533
 msgid "Selected background color"
-msgstr ""
+msgstr "Sélectionner une Couleur d'Arrière Plan"
 
 #: src/emu/ui/custui.cpp:534
 msgid "Subitem color"
-msgstr ""
+msgstr "Couleur du Sous Objet"
 
 #: src/emu/ui/custui.cpp:535 src/emu/ui/custui.cpp:630
 msgid "Clone"
-msgstr ""
+msgstr "Clone"
 
 #: src/emu/ui/custui.cpp:536
 msgid "Border"
-msgstr ""
+msgstr "Bordure"
 
 #: src/emu/ui/custui.cpp:537
 msgid "Background"
-msgstr ""
+msgstr "Arrière Plan"
 
 #: src/emu/ui/custui.cpp:538
 msgid "Dipswitch"
-msgstr ""
+msgstr "Interruption"
 
 #: src/emu/ui/custui.cpp:539
 msgid "Unavailable color"
-msgstr ""
+msgstr "Couleur Non Disponible"
 
 #: src/emu/ui/custui.cpp:540
 msgid "Slider color"
-msgstr ""
+msgstr "Couleur du Glisseur"
 
 #: src/emu/ui/custui.cpp:541
 msgid "Gfx viewer background"
-msgstr ""
+msgstr "Visualiseur GFx en Arrière Plan"
 
 #: src/emu/ui/custui.cpp:542
 msgid "Mouse over color"
-msgstr ""
+msgstr "Couleur : Sélection de la Souris"
 
 #: src/emu/ui/custui.cpp:543
 msgid "Mouse over background color"
-msgstr ""
+msgstr "Couleur du Fond de la Sélection"
 
 #: src/emu/ui/custui.cpp:544
 msgid "Mouse down color"
-msgstr ""
+msgstr "Couleur de Clic enfoncé"
 
 #: src/emu/ui/custui.cpp:545
 msgid "Mouse down background color"
-msgstr ""
+msgstr "Couleur du fond de la Sélection"
 
 #: src/emu/ui/custui.cpp:548
 msgid "Restore originals colors"
-msgstr ""
+msgstr "Restaurer les Couleurs Originales"
 
 #: src/emu/ui/custui.cpp:564
 msgid "UI Colors Settings"
-msgstr ""
+msgstr "Paramètrage des Couleurs de l'Interface"
 
 #: src/emu/ui/custui.cpp:592 src/emu/ui/selector.cpp:181
 msgid "Double click or press "
-msgstr ""
+msgstr "Double cliquer ou presser "
 
 #: src/emu/ui/custui.cpp:592
 msgid " to change the color value"
-msgstr ""
+msgstr "pour changer la valeur de la couleur"
```

Comment:
That's part of what makes string concatenation kinda bad when translating.

  OG.


On Wed, Feb 24, 2016 at 1:57 PM, Guru <notifications@github.com> wrote:

> In language/French/strings.po
> <https://github.com/mamedev/mame/pull/648#discussion_r53933882>:
>
> >
> >  #: src/emu/ui/custui.cpp:592
> >  msgid " to change the color value"
> > -msgstr ""
> > +msgstr "pour changer la valeur de la couleur"
>
> To avoid this in future, at the top of **every** .po file there should be
> a comment telling anyone who edits it to translate the English text as-is
> and not to remove leading spaces or dashes etc. because they have been put
> there on purpose and are needed when MAME selects a message and joins it
> with another, as in the above example.
> … <#268967431_>
> On Wed, Feb 24, 2016 at 7:50 PM, Vas Crabb <notifications@github.com>
> wrote: In language/French/strings.po <#648 (comment)
> <https://github.com/mamedev/mame/pull/648#discussion_r53927818>>: > > #:
> src/emu/ui/custui.cpp:592 > msgid " to change the color value" > -msgstr ""
> > +msgstr "pour changer la valeur de la couleur" This is part of a
> concatenation with the previous string - it needs the leading space — Reply
> to this email directly or view it on GitHub <
> https://github.com/mamedev/mame/pull/648/files#r53927818>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mamedev/mame/pull/648/files#r53933882>.
>